plugins/webapi/jsonmodels/info.go

```diff
@@ -50,3 +56,16 @@ type Mana struct {
 	Consensus          float64   `json:"consensus"`
 	ConsensusTimestamp time.Time `json:"consensusTimestamp"`
 }
+
+// Scheduler is the scheduler details.
+type Scheduler struct {
+	Running        bool           `json:"running"`
+	Rate           string         `json:"rate"`
+	NodeQueueSizes map[string]int `json:"nodeQueueSizes"`
```

Comment:
Is this really a meaningful metric? Potentially, the number of nodes in the queue could be rather large and is seeing the queue size per cryptic `nodeID` really helpful?

Reply:
With this we can observe the congestion of each node, and see how it works.